Sources/MessagingInApp/Gist/Managers/MessageQueueManager.swift

```diff
@@ -24,7 +24,11 @@ class MessageQueueManager {
 
     func fetchUserMessagesFromLocalStore() {
         Logger.instance.info(message: "Checking local store with \(localMessageStore.count) messages")
-        localMessageStore.forEach { message in
+        let sortedMessages = localMessageStore.sorted {
+            guard let priority0 = $0.value.priority, let priority1 = $1.value.priority else { return false }
+            return priority0 < priority1
+        }
```

Comment:
Could you add a comment to this code block explaining why we are doing this sort? Me reading the PR title and seeing this code I understand it. But future me reading this without context will probably not understand it.